operator/src/main/kubernetes/kubernetes.yml

```diff
@@ -224,8 +224,10 @@ spec:
           env:
             - name: QUARKUS_PROFILE
               value: prod
-            - name: INGRESSCONTROLLER_AZ_REPLICA_COUNT
-              value: "1"              
+            - name: KUBERNETES_MAX_CONCURRENT_REQUESTS
+              value: 100
+            - name: KUBERNETES_MAX_CONCURRENT_REQUESTS_PER_HOST
+              value: 100
```

Comment:
Why not put this in `application.properties`?

Reply:
Got it, thanks - I didn't realize these weren't the typical Quarkus/MP configs.

Reply:
I'm not sure why, but quarkus does not provide config hooks for these.  Seems like it would have become an issue for other pre-fabric8 6.4 users.